client-app/src/examples/contact/detail/DetailsPanel.scss

```diff
@@ -0,0 +1,24 @@
+.contact-details-panel {
+  &__inner {
+    height: 100%;
+    padding: var(--xh-pad-double-px);
+    overflow-y: auto;
+  }
+
+  // make this centered again
```

Comment:
I also noticed that it will trigger scrolling if wider than the detail panel width.